src/haddock/modules/topology/topoaa/defaults.yaml

```
@@ -95,6 +95,26 @@ tolerance:
   long: Percentage of allowed failures for a module to successfully complete
   group: module
   explevel: expert
+cyclicpept_dist:
+  default: 2.0
+  type: integer
+  min: 0
+  max: 9999999999999999
```

If I understand well this `cyclicpept_dist` is the distance between Nitrogen of a residue and Carbon carboxyl of an other residue ? (should be around 1.3).
So 2.00 should detect it
```suggestion
  default: 2.00
  type: float
  precision: 2
  min: 0.00
  max: 999999.99
```